src/sagemaker/workflow/pipeline.py

```diff
@@ -382,7 +382,7 @@ def start(
             sagemaker_session=self.sagemaker_session,
         )
 
-    def definition(self) -> str:
+    def definition(self) -> Dict[str, Any]:
```

Comment:
This function returns `return json.dumps(request_dict)`, which is a string ( a JSON string)
* https://www.w3schools.com/python/python_json.asp